src/components/JournalEntryDetails/JournalEntryDetails.tsx

```diff
@@ -213,9 +214,13 @@ export const JournalEntryDetails = () => {
                 )
               }
               variant={ButtonVariant.secondary}
-              onClick={onReverseEntry}
+              onClick={reverseEntryProcessing ? () => {} : onReverseEntry}
```

Comment:
No because the button visually changes when disabled (gets greyed out). I do think that it should be a no-op while processing and disabled after completed. Minor UI preference, not a super strongly held opinion.